### PR TITLE
English translation

### PR DESCRIPTION
**Pull request by passaglia**
English translation, addresses https://github.com/llm-jp/awesome-japanese-llm/issues/35

Progress: 

- [x] Preamble
- [x] テキスト生成に主に使うモデル
  - [x] 汎用
  - [x] ドメイン特化型
  - [x] （参考）英語LLMに日本語で追加学習されたモデル
- [x] 入力テキストの処理に主に使うモデル
  - [x] 汎用
  - [x] ドメイン特化型
- [x] 埋め込み (Embeddings) 作成に特化したモデル
- [x] 視覚言語モデル (Vision-Language Models)
  - [x] 画像を含むテキスト生成
  - [x] その他
- [x] （参考）日本語LLMベンチマークまとめ
- [x] （参考）各モデルの原論文
- [x] Footnotes

### CODE REVIEW

**Comment by kaisugi** [comment]
Thank you for helping me with the English translations! I was completely swamped, so I'm really happy for your support. Although this PR is still in draft status, I'll leave some comments on points I've noticed.

File: README.md [hunk 1]
```diff
@@ -214,6 +215,7 @@
 | Stable Diffusion | 2021.12.20 | CVPR 2022 | [High-Resolution Image Synthesis With Latent Diffusion Models](https://arxiv.org/abs/2112.10752) |
 | BLIP | 2022.01.28 | ICML 2022 | [BLIP: Bootstrapping Language-Image Pre-training for Unified Vision-Language Understanding and Generation](https://arxiv.org/abs/2201.12086) |
 | InstructGPT | 2022.03.04 | NeurIPS 2022 | [Training language models to follow instructions with human feedback](https://arxiv.org/abs/2203.02155) |
+| GPT-NeoX | 2022.04.14 | | (GPT-NeoX-20B: An Open-Source Autoregressive Language Model)[https://arxiv.org/pdf/2204.06745.pdf] |
```
Comment (kaisugi) [inline README.md:218]: This paper seems to have been accepted at the [BigScience Research Workshop](https://bigscience.huggingface.co/acl-2022). I would like to reveal this information to show respect to the author. 

```suggestion
| GPT-NeoX | 2022.04.14 | BigScience Research Workshop at ACL 2022 | [GPT-NeoX-20B: An Open-Source Autoregressive Language Model](https://aclanthology.org/2022.bigscience-1.9/) |
```

File: README_en.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+| LUKE | 2020.10.02 | EMNLP 2020 | [LUKE: Deep Contextualized Entity Representations with Entity-aware Self-attention](https://aclanthology.org/2020.emnlp-main.523/) |
+| CLIP | 2021.02.26 | ICML 2021 | [Learning Transferable Visual Models From Natural Language Supervision](https://arxiv.org/abs/2103.00020) |
+| RoFormer | 2021.04.20 | - | [RoFormer: Enhanced Transformer with Rotary Position Embedding](https://arxiv.org/abs/2104.09864) |
+| CLOOB | 2021.10.21 | NeurIPS 2022 | [CLOOB: Modern Hopfield Networks with InfoLOOB Outperform CLIP](https://arxiv.org/abs/2110.11316) |
+| Stable Diffusion | 2021.12.20 | CVPR 2022 | [High-Resolution Image Synthesis With Latent Diffusion Models](https://arxiv.org/abs/2112.10752) |
+| BLIP | 2022.01.28 | ICML 2022 | [BLIP: Bootstrapping Language-Image Pre-training for Unified Vision-Language Understanding and Generation](https://arxiv.org/abs/2201.12086) |
+| InstructGPT | 2022.03.04 | NeurIPS 2022 | [Training language models to follow instructions with human feedback](https://arxiv.org/abs/2203.02155) |
+| GPT-NeoX | 2022.04.14 | | (GPT-NeoX-20B: An Open-Source Autoregressive Language Model)[https://arxiv.org/pdf/2204.06745.pdf] |
```
Comment (kaisugi) [inline README_en.md:217]: same above

```suggestion
| GPT-NeoX | 2022.04.14 | BigScience Research Workshop at ACL 2022 | [GPT-NeoX-20B: An Open-Source Autoregressive Language Model](https://aclanthology.org/2022.bigscience-1.9/) |
```

File: README.md (hunk 1 above)
Comment (passaglia) [reply README.md:218]: Nice catch!

**Review by kaisugi — APPROVED** [review]
Great work!!!

**Pull request merged**